Upstream-Status: pending

automake 1.12 has deprecated support for lzma-compressed distribution archives
this patch fixes following issue with automake 1.12

| configure.ac:16: error: support for lzma-compressed distribution archives has been removed

Signed-Off-By: Nitin A Kamble <nitin.a.kamble@intel.com>
2012/05/03


Index: texinfo-4.13/configure.ac
===================================================================
--- texinfo-4.13.orig/configure.ac
+++ texinfo-4.13/configure.ac
@@ -13,7 +13,7 @@ AC_INIT([GNU Texinfo], [4.13], [bug-texi
 
 dnl Must come before AM_INIT_AUTOMAKE.
 AC_CONFIG_AUX_DIR([build-aux])
-AM_INIT_AUTOMAKE([1.10.1 readme-alpha dist-lzma])
+AM_INIT_AUTOMAKE([1.10.1 readme-alpha])
 
 # Minimum Autoconf version required.
 AC_PREREQ(2.59)
